hudi-common/src/main/java/org/apache/hudi/common/util/ReflectionUtils.java

```diff
@@ -45,7 +45,7 @@ public class ReflectionUtils {
 
   private static final Logger LOG = LoggerFactory.getLogger(ReflectionUtils.class);
 
-  private static final Map<String, Class<?>> CLAZZ_CACHE = new HashMap<>();
+  private static final Map<String, Class<?>> CLAZZ_CACHE = new ConcurrentHashMap<>();
```

Comment:
can we simplify the below code to use `computeIfAbsent` if we're using ConcurrentHashMap?

Reply:
Good advice.

Reply:
NIce advice~